tests/test_database_service.py

```diff
@@ -92,16 +92,16 @@ def setUpClass(cls) -> None:
     def test_get_institution_returns_a_label(self) -> None:
         """If IP address matches an institution, a label is returned."""
         label = TestBrowseDatabaseService.database_service.get_institution(
-            '128.84.0.0')
+            '128.84.0.0').get('label')
```

Comment:
These could test ID too or is that never used?

Reply:
Id is used, so I'll add a test.